src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -90,7 +110,39 @@ private void OnParseCommand(string value)
         }
 
         /// <summary>
-        /// Converts an hex value to a big integer
+        /// Converts a little-endian hex string to big-endian hex string
+        /// input:  ce616f7f74617e0fc4b805583af2602a238df63f
+        /// output: 0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce
+        /// </summary>
+        /// <param name="hex">Hexadecimal value to be converted</param>
+        /// <returns>Returns null when inputs is not little-endian hex string;
+        /// otherwise, returns the string that represents the converted big-endian hex string.
+        /// </returns>
+        public string? LittleEndianToBigEndian(string hex)
+        {
+            hex = hex.ToLower();
```

Comment:
```suggestion
            hex = hex.ToLowerInvariant();
```

Reply:
(or change the regex to be case non sensitive)

Reply:
we only accept `UTF-8`. No need to do this. `ToLowerInvariant` uses the `Encoding` class to apply different language based encoding for example `Windows-1252` then apply `lowercase` to the characters.

Reply:
Already modified